### PR TITLE
update: docs

### PR DESCRIPTION
**Pull request by Arrivas**
- added: tutorial
- added: aws lightsail tutorial
- added: commands
- added: bash/vim commands WIP
- edit: removed extra items on navbar

### CODE REVIEW

File: docs/guide/tutorial/aws/add-cdn-lightsail.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    2. Select the `DNS records tab` and select `Add record`
+    3. The value should be `@`, select `A record` for record type. Copy your deployed distribution link from earlier and paste it to the `Resolves to` input field. Click Save.
+![Screenshot a-name dist](./img/a-name-distribution.png)
+    4. Add a record again but this time Select `CName record` for record type and `www` for value. Copy your deployed distribution link from earlier and paste it to the `Route traffic to` input field. Click Save.
+![Screenshot cname dist](./img/c-name-distribution.png)
+    5. Test your website, to see if AWS Cloudfront (CDN) is being used, simply look for an x-cache response header. It will mention Cloudfront (e.g., Hit from Cloudfront or Miss from Cloudfront). 
+        1. To view this, go to the web-page and right click -> Inspect (or hit f12)
+        2. Select the Network tab and reload the page. 
+        3. Select a file look for the response header `x-cache` to see if Cloudfront is being used. If it is, the value will be `Hit from Cloudfront`.
+        - ![Screenshot cname dist](./img/test-cloudfront.png)
```
Comment (Keanu-Tibule) [inline docs/guide/tutorial/aws/add-cdn-lightsail.md:92]: Formatting Error. Extra indentation caused the lists and sublists to display incorrectly. Subtract an indent from the selected lines.

File: docs/guide/tutorial/aws/add-cdn-lightsail.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+### 9. Custom Domain
+If you want a custom domain for your distribution, you can add it to your distribution. 
+- *follow these steps:*
+    1. Navigate to `Networking` tab in Amazon Lightsail and select Customs Domains
+    ![Screenshot custom domain](./img/custom-domain.png)
+    2. Select the `Create Certificate`, a popup will appear, proceed adding a name and select `Continue`.
+    ![Screenshot create certificate](./img/create-certificate.png)
+    ![Screenshot create cert popup](./img/create-cert-popup.png)
+    3. Now add your origin domain, and www then click `Create Certificate`. *example*:
```
Comment (Keanu-Tibule) [inline docs/guide/tutorial/aws/add-cdn-lightsail.md:70]: Example content missing.

**Review by Keanu-Tibule — APPROVED** [review]
Slight formatting error, grammatical errors and missing content, but overall, it looks great.

File: package-lock.json
Comment (ashcslmn) [inline package-lock.json:1]: this should not be committed if no changes in package.json

**Review by ashcslmn — CHANGES_REQUESTED** [review]

**Review by ashcslmn — APPROVED** [review]

**Pull request merged**